### DISCUSSION TITLE
机器人能正常接收并发送消息，但是调用api时会遇到 Running Matcher failed.

### DISCUSSION THREAD

**Discussion by TwT-tang** [nonebot/discussions#412] [问答] [answered]
08-19 17:42:58 [INFO] nonebot_plugin_gocqhttp | [1797285720] 收到群 测试撒(451698995) 内 精神与皮囊分崩离析 意志同肌肉迎难而上(3532512015) 的消息: 测试 (75808354)
08-19 17:42:58 [SUCCESS] nonebot | OneBot V11 1797285720 | [message.group.normal]: Message 75808354 from 3532512015@[群:451698995] '测试'
08-19 17:42:58 [DEBUG] nonebot | Checking for matchers in priority 1...
08-19 17:42:58 [INFO] nonebot | Event will be handled by Matcher(type='message', module=src.plugins.msg)
08-19 17:42:58 [DEBUG] nonebot | Running Matcher(type='message', module=src.plugins.msg)
08-19 17:42:58 [DEBUG] nonebot | Running handler Dependent(call=_)
08-19 17:42:58 [INFO] nonebot | Matcher(type='message', module=src.plugins.msg) running complete
08-19 17:42:58 [ERROR] nonebot | Running Matcher(type='message', module=src.plugins.msg) failed.

api地址行代码：
anime_list=httpx.get("https://api.bilibili.com/pqc/web/timeline?types=1&before=6&after=6").json()["result"]

以及Traceback后最后一句：
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)

**Comment by yanyongyu** [comment]
报错的不是你的插件代码吗，json decode数据是空的

**Reply by TwT-tang** [reply]
是这句错了吗，我因为用异步写requests会报错就直接替换成httpx了。
anime_list=httpx.get("https://api.bilibili.com/pqc/web/timeline?types=1&before=6&after=6").json()["result"]

**Reply by yanyongyu** [reply, marked as answer]
你请求的地址没有正确返回数据

**Reply by TwT-tang** [reply]
所以是我请求的网站出问题而非我调用的语句出错是吗
